staff/staff_views_helper.py

```diff
@@ -181,23 +242,29 @@ def email_classroom(request, email_list, classroom_name):
 
 
 def get_users_from_form(form):
-    group_users = DjangoUser.objects.filter(groups__name__in=[group.name for group in form.cleaned_data.get("recipient_groups")])
-    classroom_users = DjangoUser.objects.filter(classroom__in=[classroom for classroom in form.cleaned_data.get("recipient_classrooms")])
+    group_users = DjangoUser.objects.filter(
+        groups__name__in=[
+            group.name for group in form.cleaned_data.get("recipient_groups")
+        ]
+    )
+    classroom_users = DjangoUser.objects.filter(
+        classroom__in=[
+            classroom for classroom in form.cleaned_data.get("recipient_classrooms")
```

Comment:
`[x for x in xs]` is equivalent to `list(xs)` (if not simply `xs`, if `xs` is already a list)

Reply:
67fe482fe9abd6680fe717b3f401980fb5e98518